narwhal/consensus/src/consensus.rs

```diff
@@ -214,7 +214,7 @@ pub struct Consensus<ConsensusProtocol> {
     /// Outputs the sequence of ordered certificates to the primary (for cleanup and feedback).
     tx_committed_certificates: metered_channel::Sender<Certificate>,
     /// Outputs the sequence of ordered certificates to the application layer.
-    tx_sequence: metered_channel::Sender<ConsensusOutput>,
+    tx_sequence: metered_channel::Sender<Box<CommittedSubDag>>,
```

Comment:
I don't think we actually need `Box` here - note that `std::Vec` already puts all it's content on the heap, and with this in mind `CommittedSubDag` is not a very large data structure as it only contains Certificate + Vec(and we seem to be ok to sending Certificate across the channels)